lib/slack-ruby-bot/rspec/support/slack-ruby-bot/it_behaves_like_a_slack_bot.rb

```diff
@@ -4,12 +4,13 @@
   context 'not configured' do
     before do
       @slack_api_token = ENV.delete('SLACK_API_TOKEN')
+      SlackRubyBot.configure { |config| config.token = nil }
```

Comment:
Implement or use `reset` for the configuration and reset all these tokens and ENV and restore ENV afterwards in a support/....rb. Like In https://github.com/slack-ruby/slack-ruby-client/blob/7c93cec20b87b38388b8afe092ca5d6adf71e51e/spec/support/token.rb, so we don't have to do it all over the place and always start with a blank configuration.

Reply:
Looks like these are already in:
- https://github.com/slack-ruby/slack-ruby-bot/blob/master/lib/slack-ruby-bot/rspec/support/slack_ruby_bot_configure.rb
- https://github.com/slack-ruby/slack-ruby-bot/blob/master/lib/slack-ruby-bot/rspec/support/slack_api_key.rb

I'll change to use those!